platform/ui/src/index.js

```diff
@@ -52,6 +52,11 @@ import SnackbarProvider, {
   useSnackbarContext,
   withSnackbar,
 } from './utils/SnackbarProvider';
+import ModalProvider, {
+  useModalContext,
```

Comment:
I realize this matches `useSnackbarContext`, but would it make more sense if this was simple `useModal`? Or am I misconstruing what this is?

Reply:
It could be renamed for sure, this is just a hook, a component calling a useContext will always rerender when the context value changes